server/queues/processors/NotificationsProcessor.ts

```diff
@@ -99,12 +100,30 @@ export default class NotificationsProcessor extends BaseProcessor {
         );
       }
     }
+
+    // send notifs to mentioned users
+    const mentions = parseMentions(document);
+    for (const mention of mentions) {
+      const [recipient, actor] = await Promise.all([
+        User.findByPk(mention["data-id"]),
```

Comment:
```suggestion
        User.findByPk(mention.id),
```

This will be like this when above suggestions are made